### PR TITLE
feat: [SRM-15016]: adding tabs in the configurations sections for srm module

### PR DESCRIPTION
**Pull request by harshilbits**
### Summary
1. Adding tabs in the configurations sections for srm module

#### Screenshots
New Note

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx [hunk 1]
```diff
@@ -188,107 +193,115 @@ function Service(
...
+                </Text>
+              ) : null}
+              {isHealthSrcSectionHidden ? null : (
+                <HealthSourceTableContainer
+                  healthSourceListFromAPI={initialValues.sources?.healthSources}
+                  serviceFormFormik={formik}
+                  isTemplate={isTemplate}
+                  expressions={expressions}
+                  onSave={
+                    /* istanbul ignore next */ data => {
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx:264]: why we are ignoring this callback?

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx [hunk 2]
```diff
@@ -188,107 +193,115 @@ function Service(
...
+                    /* istanbul ignore next */ () => {
+                      openChangeSourceDrawer({ formik, onSuccessChangeSource })
+                    }
+                  }
+                  value={formik.values?.sources?.changeSources}
+                  onSuccess={onSuccessChangeSource}
+                />
+              )}
+              {isNotificationsSectionHidden ? null : (
+                <>
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx:294]: please remove this fragment

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx [hunk 3]
```diff
@@ -188,107 +193,115 @@ function Service(
...
+                  }
+                />
+              )}
+              {isChangeSrcSectionHidden ? null : (
+                <ChangeSourceTableContainer
+                  onEdit={values => {
+                    showDrawer({ ...values, hideDrawer })
+                  }}
+                  onAddNewChangeSource={
+                    /* istanbul ignore next */ () => {
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx:285]: why we are ignoring this callback?

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/CommonMonitoredServiceConfigurations/__tests__/CommonMonitoredServiceConfigurations.test.tsx [hunk 4]
```diff
@@ -126,16 +126,16 @@ describe('CommonMonitoredServiceConfigurations', () => {
...
+    const changeSourceTab = queryByText('cv.navLinks.adminSideNavLinks.activitySources')
 
     expect(healthSourceTab).not.toBeInTheDocument()
     expect(changeSourceTab).not.toBeInTheDocument()
   })
 
   test('should render', async () => {
     const { container, getByText, queryByText } = render(<WrapperComponent {...props} />)
-    userEvent.click(queryByText('changeSource')!)
+    userEvent.click(queryByText('cv.navLinks.adminSideNavLinks.activitySources')!)
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/CommonMonitoredServiceConfigurations/__tests__/CommonMonitoredServiceConfigurations.test.tsx:138]: use `await userEvent.click(...)` this is mandate in updated version.

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx (hunk 1 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx]: removed

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx (hunk 2 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx]: done.

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx (hunk 3 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.tsx]: removed

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/CommonMonitoredServiceConfigurations/__tests__/CommonMonitoredServiceConfigurations.test.tsx (hunk 4 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/CommonMonitoredServiceConfigurations/__tests__/CommonMonitoredServiceConfigurations.test.tsx]: done.

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/CommonMonitoredServiceConfigurations/__tests__/CommonMonitoredServiceConfigurations.test.tsx [hunk 5]
```diff
@@ -126,16 +126,16 @@ describe('CommonMonitoredServiceConfigurations', () => {
...
+    const changeSourceTab = queryByText('cv.navLinks.adminSideNavLinks.activitySources')
 
     expect(healthSourceTab).not.toBeInTheDocument()
     expect(changeSourceTab).not.toBeInTheDocument()
   })
 
   test('should render', async () => {
     const { container, getByText, queryByText } = render(<WrapperComponent {...props} />)
-    userEvent.click(queryByText('changeSource')!)
+    await userEvent.click(queryByText('cv.navLinks.adminSideNavLinks.activitySources')!)
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/CommonMonitoredServiceConfigurations/__tests__/CommonMonitoredServiceConfigurations.test.tsx:138]: NIT, if you are sure that an element will present in the DOM, then use `getBy...`. To assert absence of an element use `queryBy...`

Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/components/CommonMonitoredServiceConfigurations/__tests__/CommonMonitoredServiceConfigurations.test.tsx:138]: sure.

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**